docs_user/assemblies/assembly_migrating-databases-to-the-control-plane.adoc

```diff
@@ -16,9 +28,9 @@ include::../modules/proc_configuring-a-ceph-backend.adoc[leveloffset=+1]
 
 include::../modules/proc_stopping-openstack-services.adoc[leveloffset=+1]
 
-include::../modules/proc_migrating-databases-to-mariadb-instances.adoc[leveloffset=+1]
+include::../modules/proc_migrating-databases-to-mariadb-instances.adoc[@, leveloffset=+1]
```

Comment:
What is this symbol for? 
Same question on the link below.

Reply:
an error on my part I thought it was needed, so the vars pass to the included files, I'll remove that